gsrs-spring-boot-autoconfigure/src/main/java/gsrs/controller/AbstractImportSupportingGsrsEntityController.java

```diff
@@ -679,6 +679,34 @@ public ResponseEntity<Object> updateImport(@RequestBody JsonNode updatedJson,
         return new ResponseEntity<>(GsrsControllerUtil.enhanceWithView(itmd, queryParameters), HttpStatus.OK);
     }
 
+    @hasAdminRole
+    @PostGsrsRestApiMapping(value = {"/stagingArea"})
+    public ResponseEntity<Object> createEntity(@RequestBody String newEntity,
+                                               @RequestParam Map<String, String> queryParameters) throws Exception {
+        log.trace("in createEntity");
+        String entityType = queryParameters.get("entityType");//type of domain object to create
+        Objects.requireNonNull(entityType, "Must supply entityType (class of object to create)");
+        String adapterName = queryParameters.get("adapter");
```

Comment:
In this particular case I think we're expecting they're POSTing the JSON directly, so they shouldn't need an adapter?

Reply:
Or if it is an adapter, it should be essentially a no-op one?